src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/CalibrationData.cs

```diff
@@ -1,102 +1,99 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System.IO;
-using UnityEngine;
-
-namespace Microsoft.MixedReality.SpectatorView
-{
-    /// <summary>
-    /// Used for loading/applying camera intrinsics and camera extrinsics obtained through Spectator View's default calibration process.
-    /// </summary>
-    public class CalibrationData : ICalibrationData
-    {
-        private CalculatedCameraIntrinsics cameraIntrinsics;
-        private CalculatedCameraExtrinsics cameraExtrinsics;
-
-        public CalibrationData(string cameraIntrinsicsPath, string cameraExtrinsicsPath)
-        {
-            if (File.Exists(cameraIntrinsicsPath) &&
-                File.Exists(cameraExtrinsicsPath))
-            {
-                cameraIntrinsics = CalibrationDataHelper.LoadCameraIntrinsics(cameraIntrinsicsPath);
-                cameraExtrinsics = CalibrationDataHelper.LoadCameraExtrinsics(cameraExtrinsicsPath);
-
-                if (cameraIntrinsics == null ||
-                    cameraExtrinsics == null)
-                {
-                    Debug.LogError($"Failed to load cameara intrinsics/extrinscs: {cameraIntrinsicsPath}, {cameraExtrinsicsPath}");
-                }
-            }
-            else
-            {
-                Debug.LogError($"Invalid paths provided for camera intrinsics/extrinsics: {cameraIntrinsicsPath}, {cameraExtrinsicsPath}");
-            }
-        }
-
-        public CalibrationData(CalculatedCameraIntrinsics intrinsics, CalculatedCameraExtrinsics extrinsics)
-        {
-            cameraIntrinsics = intrinsics;
-            cameraExtrinsics = extrinsics;
-        }
-
-        /// <inheritdoc />
-        public void SetUnityCameraExtrinstics(Transform cameraTransform)
-        {
-            Matrix4x4 headFromCamera = cameraExtrinsics.ViewFromWorld;
-            cameraTransform.transform.localPosition = cameraExtrinsics.ViewFromWorld.GetColumn(3);
-            cameraTransform.transform.localRotation = Quaternion.LookRotation(cameraExtrinsics.ViewFromWorld.GetColumn(2), cameraExtrinsics.ViewFromWorld.GetColumn(1));
-
-            // Magic offset from Unity's underlying coordinate frame (WorldManager.GetNativeISpatialCoordinateSystemPtr()) and the head pose used for the camera.
-            // Poses are sent in the coordinate frame space because the Unity camera position uses prediction.
-            cameraTransform.localPosition += new Vector3(0f, 0.08f, 0.08f);

```

Comment:
Did we determine this actually changed? What version of unity are we using?

Reply:
I moved this from the editor side to the HoloLens side. That way the offset of the HoloLens isn't baked into every CalibrationData, so CalibrationData can be reused between HoloLens+DSLR and AzureKinect.  Now this offset is applied in the HolographiCamera app before it sends the calibration data back to the compositor.

---
In reply to: [384166417](https://github.com/microsoft/MixedReality-SpectatorView/pull/327#discussion_r384166417) [](ancestors = 384166417)